src/components/utils.ts

```diff
@@ -43,7 +46,7 @@ export async function handleInstall({
   if (installPath === 'import') {
     const { filePaths } = await showOpenDialog({
       buttonLabel: t('gamepage:box.choose'),
-      properties: ['gamepage:openDirectory'],
+      properties: ['openDirectory'],
```

Comment:
What happened here?

Reply:
This is a bug. If you look at it, we should have a property but we are using a translation key. I fixed this long time ago but I think got back on some rebase.
This bug make it impossible to select a folder when installing from the library.